pygmt/src/pygmtlogo.py

```diff
@@ -0,0 +1,275 @@
+"""
+pygmtlogo - Create and plot the PyGMT logo.
+The design of the logo is kindly provided by `@sfrooti <https://github.com/sfrooti>`_.
+"""
+
+from pathlib import Path
+
+import pygmt
+
+
+def pygmtlogo(  # noqa: PLR0915
+    self,
+    blackwhite=False,
+    darkmode=True,
+    hexshape=False,
+    wordmark=True,
+    orientation="horizontal",  # "horizontal" | "vertical"
+    position="jRT+o0.1c+w5c",  # -> use position parameter of Figure.image
+    box=None,  # -> use box parameter of Figure.image
+):
```

Comment:
Probably we also need the parameters `region`, `projection`, `panel`, `perspective`, and `transparency` of `Figure.image`.

Reply:
Added in commit https://github.com/GenericMappingTools/pygmt/pull/3849/commits/1a2ca98c535db85bff5b1bd3d307ce529c16aaf9.